src/index.html

```diff
@@ -2,16 +2,170 @@
+      <fieldset class="fieldsetstyle">
+        <legend>Personal information</legend>
+        <label class="form-field" for="surname">
+          Surname:
+          <input
+          id="surname"
+          type="text"
+          name="surname"
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label class="form-field" for="name">
+          Name:
+          <input
+          id="name"
+          type="text"
+          name="name"
+          required
+          >
+        </label>
+        <label class="form-field" for="age">
+          How old are You?
+          <input
+          id="age"
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+          required
+          >
+        </label>
+        <label class="form-field" for="dateOfBirth">
+          Full date of birth:
+          <input
+          id="dateOfBirth"
+          type="date"
+          name="dateOfBirth"
+          >
+        </label>
+        <label class="form-field" for="accepTerm">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="accepTerm"
+          id="accepTerm"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldsetstyle">
+        <legend>Registration</legend>
+        <label class="form-field" for="email">
+          E-mail:
+          <input
+          id="email"
+          type="email" 
+          name="email"
+          placeholder="email@example.com"
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label class="form-field" for="password">
+          Password:
+          <input
+          id="password"
+          type="password" 
+          name="password"
+          autocomplete="off"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldsetstyle">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field" for="fact">
+          Do you love cats?
+          <input
+          id="fact"
+          type="radio"
+          name="interestingFact"
+          >
+          Yes
+          <input
+          id="facts"
+          type="radio"
+          name="interestingFact"
+          >
+          No
+        </label>
+        <label class="form-field" for="color">
+          What is your favorite color?
+          <input
+          id="color"
+          type="color"
+          name="color"
+          >
+        </label>
+        <label class="form-field" for="bedTime">
+          What time do you go to bed?
+          <input
+          id="bedTime"
+          type="time"
+          name="timeToBed"
+          step="2"
+          >
+        </label>
+        <label class="form-field" for="car">
+          What are your favorite brand of cars?
+          <select
+          name="cars"
+          id="car"
+          multiple>
```

Comment:
```suggestion
          <select
            name="cars"
            id="car"
            multiple
          >
```
Closing tag should be just under the opening